src/main/java/ru/mail/dimaushenko/repository/connectionPool/ConnectionPool.java

```diff
@@ -0,0 +1,41 @@
+package ru.mail.dimaushenko.repository.connectionPool;
```

Comment:
the package should be at the lower case and separate by a dot. No need to create a package for class with the same name